semcore/mini-chart/src/component/trend/Bar.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import createComponent, { ComponentType, assignProps, Root, sstyled } from '@semcore/core';
+import resolveColorEnhance from '@semcore/utils/lib/enhances/resolveColorEnhance';
+import { Trend, CommonTrendProps } from './Trend';
+import { Box } from '@semcore/flex-box';
+
+import style from '../skeleton/skeleton.shadow.css';
+
+type BarItem = {
+  /**
+   * Value
+   */
+  value: number;
+  /**
+   * Color of value
+   */
+  color?: string;
+};
+
+type TrendBarProps = CommonTrendProps & {
+  /**
+   * Data to bar chart
+   */
+  data: BarItem[];
+};
+
+type Enhances = {
+  resolveColor: ReturnType<typeof resolveColorEnhance>;
+  isHistogram?: true;
+};
+
+class TrendBarRoot extends Trend<TrendBarProps, Enhances> {
+  static enhance = [resolveColorEnhance()];
+
+  static style = style;
+
+  static defaultProps = {
+    animate: true,
+  };
+
+  get defaultData(): BarItem[] {
+    return [{ value: 20 }, { value: 80 }, { value: 45 }, { value: 10 }];
+  }
+
+  get data(): BarItem[] {
+    const { data, isLoading } = this.asProps;
+
+    if (isLoading) {
+      return this.defaultData;
+    }
+
+    return data;
+  }
+
+  render() {
+    const STrendBar = Root;
+    const { styles, resolveColor, isHistogram, animate, isLoading } = this.asProps;
+    const step = this.defaultWidth / this.data.length;
+
+    return sstyled(styles)(
+      <STrendBar render={Box} ref={this.containerRef} __excludeProps={['data']}>
+        <svg
+          width='100%'
+          height='100%'
+          viewBox={`0 0 ${this.svgWidth} ${this.svgHeight}`}
+          xmlns='http://www.w3.org/2000/svg'
```

Comment:
Do we really need `xmlns` prop?

Reply:
just copy-paste from svg from figma, removed